packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -67,11 +77,35 @@
 
 <script>
 import parseISO from 'date-fns/parseISO';
-import lightFormat from 'date-fns/format';
+import format from 'date-fns/format';
+import lightFormat from 'date-fns/lightFormat';
 
 import EmptyStamp from './images/stamp-empty-15.svg';
 import FullStamp from './images/stamp-full-15.svg';
 
+import '../MakeTextAccessible';
+
+const getDateFnsLocale = locale => {
+    switch (locale) {
+        case 'da-DK':
+            return import(/* webpackChunkName */ 'date-fns/locale/da');
```

Comment:
You can either remove the `webpackChunkName` comment (I don't believe it's required), or pass it something like `webpackChunkName: "locale-[request]"` to generate a readable name.